test/commons/text/accessible-text.js

```diff
@@ -3208,6 +3208,17 @@ describe('text.accessibleTextVirtual', function() {
       assert.equal(accessibleText(target), 'Country of origin: United States');
     });
 
+    it('passes test 160', function() {
```

Comment:
Sorry missed this one last time. These tests are specifically acceptance tests taken from the [AccName 1.1 Testable Statements](https://www.w3.org/wiki/AccName_1.1_Testable_Statements#Name_test_case_539). Since this exact case doesn't appear to be covered by those tests, we'll want to add this test [above this entire section](https://github.com/dequelabs/axe-core/blob/70ecb6dce4fd8b5bda5b75eb2ce4696237cc4628/test/commons/text/accessible-text.js#L1426) as it's own test. That way we don't imply it's an acceptance test.